config/v1/types_node.go

```diff
@@ -42,6 +42,11 @@ type NodeSpec struct {
 	// the status and corresponding reaction of the cluster
 	// +optional
 	WorkerLatencyProfile WorkerLatencyProfileType `json:"workerLatencyProfile,omitempty"`
+
+	// EventedPleg enables event based PLEG between the kubelet and the CRI-O
```

Comment:
I think we want to expand this comment to provide more detail.

Questions I would be asking would be:
* What are the valid values?
* What happens if i didn't specify any value? Is this behaviour likely to change over time?

Nit, the comment should start with the json tag version of the name not the Go field version
So I'd be expecting something more along the lines of:
```suggestion
// eventedPLEG enables event based PLEG between the kubelet and the CRI-O.
// Valid values are `Enabled`, `Disabled` and omitted.
// When omitted, this means no opinion and the platform is left to choose a reasonable default
// which is subject to change over time.
// The current default is `Disabled`.
```

Also, is it worthing rather than saying `CRI-O`, spelling that out in human readable prose?

I assume a user will know what PLEG is if they are turning this on?

Reply:
Updated as suggested along with a reference to the KEP !!